packages/opentelemetry-plugin-mysql/src/mysql.ts

```diff
@@ -0,0 +1,266 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BasePlugin } from '@opentelemetry/core';
+import { CanonicalCode, Span, SpanKind } from '@opentelemetry/types';
+import * as mysqlTypes from 'mysql';
+import * as shimmer from 'shimmer';
+import { AttributeNames } from './enums';
+import { getConnectionAttributes, getSpanName } from './utils';
+
+export class MysqlPlugin extends BasePlugin<typeof mysqlTypes> {
+  readonly moduleName = 'mysql';
+  readonly supportedVersions = ['2.*'];
```

Comment:
Could you please add this supported version in mysql README similar to https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-plugin-redis#supported-versions?